electron-app/src/views/settings/AccountingSettings.vue

```diff
@@ -15,43 +10,41 @@
               class="settings-accounting__crypto2crypto"
               label="Take into account crypto to crypto trades"
               color="primary"
+              :success-messages="settingsMessages['crypto2crypto'].success"
```

Comment:
I am curious, doesn't `settingsMessages.crypto2crypto.success` work? is there any specific reason you followed this approach?

Reply:
I think it would work. If I remember I did it like this because this is how it's accessed ~below~ in Settings.vue (where you need to dynamically access the index). I was/am thinking of pulling this functionality out into a custom component so that we can use it in other places in the app as well (anywhere you might do CRUD). If it's an anti-pattern or ugly I'm happy to change.

Reply:
Yes in a new component it would make sense. I was also thinking about the component approach. It makes more sense than repeating this everywhere.

Reply:
The downside is we would need to do this for every type of form element we might use, so that's a lot of things to write a wrapper for... and then I never know how many of the "default" vuetify props I should pass through. Is there a way for me to pass everything through but override specific ones instead of doing them one by one?